src/uwtools/tests/config/test_support.py

```diff
@@ -85,14 +85,9 @@ def loader(self):
         return yaml.SafeLoader("")
 
     # These tests bypass YAML parsing, constructing nodes with explicit string values. They then
-    # demonstrate that those nodes' convert() methods return representations in type type specified
+    # demonstrate that those nodes' convert() methods return representations in the type specified
     # by the tag.
 
-    def test_bool_bad(self, loader):
-        ts = support.UWYAMLConvert(loader, yaml.ScalarNode(tag="!bool", value="foo"))
-        with raises(KeyError):
-            ts.convert()
-
```

Comment:
Since there are no values in Python that can't be converted into a `bool` object, I removed this test.